src/content/ja/2021/capabilities.md

```diff
@@ -0,0 +1,441 @@
+---
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+title: ケイパビリティ
+description: 2021年版Web Almanacの「ケイパビリティ」の章では、Webアプリケーションにハードウェアインタフェースへのアクセスを提供し、Webベースの生産性アプリケーションを強化する、全く新しい強力なWebプラットフォームAPIをカバーしています。
+authors: [christianliebel]
+reviewers: [tomayac, hemanth]
+analysts: [tomayac]
+editors: [tunetheweb]
+translators: [ksakae1216]
+results: https://docs.google.com/spreadsheets/d/1b4moteB9EiLYkH1Ln9qfi1tnU-E4N2UQ87uayWytDKw/
+christianliebel_bio: Christian Liebel は、<a hreflang="en" href="https://thinktecture.com">Thinktecture</a> のコンサルタントで、さまざまなビジネス分野のクライアントが優れた Web アプリケーションを実装できるように支援しています。Microsoft MVP for Developer Technologies、Google GDE for Web/Capabilities and Angularを取得しており、W3C Web Applications Working Groupに参加しています。
```

Comment:
Do the `for` and `and`s need translated?

Reply:
Thanks, I translated.